modules/local/combinegvcfs.nf

```diff
@@ -0,0 +1,65 @@
+process COMBINEGVCFS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gatk4:4.5.0.0--py36hdfd78af_0':
+        'biocontainers/gatk4:4.5.0.0--py36hdfd78af_0' }"
```

Comment:
We might want to move this logic in module.config.

Reply:
I think it is more "nfcore-like" to keep the container descriptions in the modules themselves. For now, however I tried the nf-core containers and they don't work on juno, so we have to keep them in modules.config

Reply:
Ok, thanks for sharing.  It's weird ... I will have a closer look at this. You can merge anyway.